tests/ipex/test_pipelines.py

```diff
@@ -56,7 +56,6 @@ class PipelinesIntegrationTest(unittest.TestCase):
         "gpt2",
         "gpt_neo",
         "gpt_neox",
-        "llama",
```

Comment:
why removing the pipeline test for llama models ?

Reply:
There are limitation for head_size of IAKV which is >=64.